docs/content/querying/dimensionspecs.md

```diff
@@ -252,3 +252,16 @@ A null dimension value can be mapped to a specific value by specifying the empty
 This allows distinguishing between a null dimension and a lookup resulting in a null.
 For example, specifying `{"":"bar","bat":"baz"}` with dimension values `[null, "foo", "bat"]` and replacing missing values with `"oof"` will yield results of `["bar", "oof", "baz"]`.
 Omitting the empty string key will cause the missing value to take over. For example, specifying `{"bat":"baz"}` with dimension values `[null, "foo", "bat"]` and replacing missing values with `"oof"` will yield results of `["oof", "oof", "baz"]`.
+
+### Filtering DimensionSpecs
+These are only valid for multi-valued dimensions. They take a delegate DimensionSpec and a filtering criteria, multiple values list of dimension is filtered as per given criteria.
+
+Following filtered dimension spec acts as a whiltelist or blacklist for values as per the configuration.
```

Comment:
I think one or two examples of how to use this optimization will help clear up the confusion.